### ISSUE TITLE
Implement Graphics::Impl::DevicePixelRatio() properly

### ISSUE THREAD

**Issue by Drigax** [BabylonJS/BabylonNative#625]
In UWP, Due to the constraints of the platform, we need a way to dispatch calls to the UI thread from the worker threads that handles the graphics work and Javascript callbacks.

Currently, I plan to add this as a UWP-only member of our configuration struct, which would be cached and accessible from the Graphics::Impl object.

In Linux, we need a reference to our X11 Display in order to use the XLib RandR API to determine the screen pixel scaling.

Since both work items are dependent on #620, I'll have to implment this after the contract is in

**Comment by thomlucc** [comment]
Not needed separately

**Comment by bghgary** [comment]
This work is not done yet.

**Comment by bghgary** [comment]
Closing in favor of #1016.
